panel/src/api/users.js

```diff
@@ -16,8 +16,11 @@ export default (api) => ({
 	async changeName(id, name) {
 		return api.patch("users/" + id + "/name", { name: name });
 	},
-	async changePassword(id, password) {
-		return api.patch("users/" + id + "/password", { password: password });
+	async changePassword(id, currentPassword, password) {
```

Comment:
`Suggestion:` Add the new parameter last so that old code does not pass new passwords now as `currentPassword`.

Reply:
I thought about it, but decided against it because with order-based arguments, we would never be able to flip the order later without making the mess even bigger. So we would forever have that order that doesn't match the "intuitive" order.

Given that there is only one place in the entire Panel codebase that called this method and uses in plugins are unlikely/not super common, I think documenting it as a breaking change is a more sensible solution overall.

Reply:
Fair. One more thought though: is this really the intuitive order? I'd think that id and new password are more linked as they both belong to the same user (the one updated) while current password can belong to that user but also another (current) user. That's why my thought would be the other two go first together, and only then current password. If you still think the suggested order is the more intuitive, feel free to resolve this though. It's just a suggestion.